src/Implement/UnitTestOperateData.cpp

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) 2021, 鍾淯丞, 周杰仕, 林仁鴻. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+
+#include "OperateData.h"
```

Comment:
Please add unit test.

Your unit test will use the OperateData in a way that it is usually used.
i.e. set the header and body, then call operator[] on it.